### PR TITLE
Add website content guidance

### PR DESCRIPTION
**Pull request by robmarcer**

### CODE REVIEW

File: design/index.md [hunk 1]
```diff
@@ -40,4 +40,11 @@ Other tools included in the Creative Cloud suite that may be useful include:
 
 ### Blender ([more info](https://blender.org))
 
-Blender is a free and open-source 3D computer graphics software toolset. It can be used to create static renders of 3d assets, or entire 2d & 3d animations. 
\ No newline at end of file
+Blender is a free and open-source 3D computer graphics software toolset. It can be used to create static renders of 3d assets, or entire 2d & 3d animations. 
+
+### Websites
```
Comment (joepavitt) [inline design/index.md:45]: When referring to "websites" - is this just our own FlowForge website, or encompassing the app.flowforge.com too?

Comment (robmarcer) [reply design/index.md:45]: I don't think this should be used for the app, I think the best practices will not be the same. What do you think?

Comment (joepavitt) [reply design/index.md:45]: I agree - just wanted to clarify

**Review by joepavitt — APPROVED** [review]

**Comment by joepavitt** [comment]
Will wait for a @ZJvandeWeg nod of approval before merrging

**Pull request merged**